app/api/aggregator.ts

```
@@ -315,3 +315,26 @@ export async function updateTransactionDetails({
+/**
+ * Fetches the list of supported tokens from the aggregator API
+ * @returns {Promise<any[]>} Array of supported tokens from the API
+ * @throws {Error} If the API request fails
+ */
+export const fetchTokens = async (): Promise<any[]> => {
+  try {
+    const response = await axios.get(`${AGGREGATOR_URL}/tokens`);
+    // Handle the API response structure: { status, message, data }
+    if (
+      response.data &&
+      response.data.data &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)